tests/ios/xcconfig/WithDefinesSwift/withDefines.swift

```diff
@@ -0,0 +1,22 @@
+enum WithDefinesSwift {
+  func foo() {
+#if !MACRO_H
+ thisShouldNotBeCompiled
```

Comment:
Same as above but `#error("MACRO_H should be set via merged xcconfig")`

Reply:
I'll update!